enclave-runner/src/usercalls/mod.rs

```diff
@@ -346,36 +381,221 @@ impl EnclaveState {
             fds: Mutex::new(fds),
             last_fd,
             exiting: AtomicBool::new(false),
-            usercall_ext : usercall_ext,
+            usercall_ext,
         })
     }
 
+    fn create_worker_threads(num_of_worker_threads: usize,
```

Comment:
It doesn't really make sense this is on `impl EnclaveState`. Move this function so it's defined inside `setup_and_start_work`